Downsample/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+project(Downsample)
+
+cmake_minimum_required(VERSION 3.3.2 FATAL_ERROR)
+
+set(CMAKE_CXX_STANDARD 11)
+
+find_package(Zivid COMPONENTS Core REQUIRED)
+
+add_custom_target(
+    SampleDependencies
+    COMMAND ${CMAKE_COMMAND} -E copy_directory
+            $ENV{ZIVID_INSTALL_FOLDER}/$<$<CONFIG:Debug>:bin_debug>$<$<CONFIG:Release>:bin>/plugins/
+            ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_CFG_INTDIR}/plugins
+    COMMAND ${CMAKE_COMMAND} -E copy
+            ${CMAKE_CURRENT_SOURCE_DIR}/../Zivid3D.zdf
+            ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_CFG_INTDIR}
+)
+
+#find_package(Eigen3 REQUIRED)
+SET( EIGEN3_INCLUDE_DIR "$ENV{EIGEN3_INCLUDE_DIR}" )
+IF( NOT EIGEN3_INCLUDE_DIR )
+    MESSAGE( FATAL_ERROR "Please point the environment variable EIGEN3_INCLUDE_DIR to the include directory of your Eigen3 installation.")
+ENDIF()
+INCLUDE_DIRECTORIES ( "${EIGEN3_INCLUDE_DIR}" )
```

Comment:
I couldn't make this work so I had to manually import Eigen headers path. I think my cmake has the worng path and I don't know how to change it.

Reply:
I tried to build one version and failed. Then I just downloaded another version which I am using now.

Reply:
No need to find the package because the tutorial will have:
cmake DEIGEN3_INCLUDE_DIR="path_to_eigen" ..
It will also tell exactly which version is needed and where to clone it.